### PR TITLE
Giftset

### PR DESCRIPTION
**Pull request by Albinka-811**

### CODE REVIEW

File: src/styles/components/tab.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.tab {
+  right: 0;
+  position: absolute;
+  width: 88px;
+  height: 460px;
+}
+
+.tablink {
```
Comment (Anatolych97) [inline src/styles/components/tab.scss]: Use & in styles

File: index.html [hunk 2]
```diff
@@ -213,6 +213,53 @@ <h5 class="slider-products-row-content-info__name">REVO Origin</h5>
...
+                    1
+                </button>
+                <button class="tablink" onclick="openGiftSet(event, 'Arabic')">
+                    2
+                </button>
+                <button class="tablink " onclick="openGiftSet(event, 'Nescafe')">
+                    3
+                </button>
+            </div>
+            <img class="giftset-depiction-photo" src="src/assets/images/set.png" alt="">
```
Comment (Anatolych97) [inline index.html:233]: Create related function

File: src/styles/components/tab.scss (hunk 1 above)
Comment (Albinka-811) [reply src/styles/components/tab.scss]: correct

File: index.html (hunk 2 above)
Comment (Albinka-811) [reply index.html:233]: fixed

**Pull request merged**